Makefile

```diff
@@ -89,6 +89,6 @@ reindex-solr:
 	psql openlibrary -t -c 'select key from thing' | sed 's/ *//' | grep '^/books/' | PYTHONPATH=$(PWD) xargs python openlibrary/solr/update_work.py -s http://0.0.0.0/ -c conf/openlibrary.yml --data-provider=legacy
 	psql openlibrary -t -c 'select key from thing' | sed 's/ *//' | grep '^/authors/' | PYTHONPATH=$(PWD) xargs python openlibrary/solr/update_work.py -s http://0.0.0.0/ -c conf/openlibrary.yml --data-provider=legacy
 
-test:
+test: css
 	npm install && npm test
```

Comment:
I don't think `npm install` should be here, it takes a long time to install each time (I tried it in my dev environment (Docker) where I frequently run `make test`). Adding the install to `travis.yml` and the `Docker.oldev` would be more appropriate, I think.

Reply:
Thanks for the feedback.
Happily can move this out of the Makefile and run it in CI via other means

Reply:
Looks like this is unblocked now. Yay!